src/Orsak/Effect.fs

```diff
@@ -343,7 +343,7 @@ type Effect =
     ///         Orsak.Effect.Create(fun (e: #MyEffectProvider) -> e.Effect.DoEffect(x))
     ///   </code>
     /// </example>
-    static member Create(f: 'a -> Task<'b>, [<Optional>] _medium: byte) : Effect<'a, 'b, _> =
+    static member Create(f: 'a -> Task<'b>) : Effect<'a, 'b, _> =
```

Comment:
We need that parameter for overload resolution, or we need to create a module with a type extension in order for the overload resolution to work